values.yaml

```diff
@@ -6,11 +6,16 @@ image:
 deployment:
   replicas: 1
 #
+service:
```

Comment:
Should be commented out by default

Reply:
I am not sure to understand: if you comment the service section by default, the service won't have any type. Could you elaborate ?

Reply:
The service should be commented out, as it should not be a requirement to set.

Please see https://github.com/containous/traefik-helm-chart/commit/08996411524c74d28895541a893116e5bc28eed0 as an example of what I mean about setting the default value if not set.

Reply:
>Do you want me to add an if/else condition that is more resilient 

Yes please, that is preferable. Ideally we don't want to mandate more values in the values.yaml than is needed, as it ties our hands for backwards compatibility.

Reply:
I tried to make it clean but the way helm implement the 'and' made me stack ifs.